jaggery-modules/rxt/module/scripts/asset/asset.js

```diff
@@ -1667,6 +1711,49 @@ var asset = {};
         return asset;
     };
 
+    /***
+     * This method is used get the Current Lifecycle State Duration Meta Data.
+     * @param asset
+     * @returns {{}} json object that contains required metadata
+     */
+    AssetManager.prototype.getLCStateDurationDetails = function (asset) {
+        var details = {};
```

Comment:
Thank you Sameera,
This entire code block has been refactored with new time in lifecycle feature.